cmd/release-controller/main.go

```diff
@@ -200,7 +200,7 @@ func (o *options) Run() error {
 	execReleaseInfo := NewExecReleaseInfo(client, config, o.JobNamespace, fmt.Sprintf("%s", releaseNamespace), imageCache.Get)
 	releaseInfo := NewCachingReleaseInfo(execReleaseInfo, 64*1024*1024)
 
-	execReleaseFiles := NewExecReleaseFiles(client, config, o.JobNamespace, fmt.Sprintf("%s", releaseNamespace), imageCache.Get)
+	execReleaseFiles := NewExecReleaseFiles(client, config, o.JobNamespace, fmt.Sprintf("%s", releaseNamespace), fmt.Sprintf("%s", releaseNamespace), imageCache.Get)
```

Comment:
I see it's a prexisting pattern, but what's this `Sprintf` stuff about?  Isn't `releaseNamespace` already a `string`?

```console
$ git describe --always --dirty
03d8c65c
$ git --no-pager grep ReleaseNamespaces
cmd/release-controller/main.go:	ReleaseNamespaces []string
cmd/release-controller/main.go:	flag.StringSliceVar(&opt.ReleaseNamespaces, "release-namespace", opt.ReleaseNamespaces, "The namespace where the source image streams are located and where releases will be published to.")
cmd/release-controller/main.go:	if len(o.ReleaseNamespaces) == 0 {
cmd/release-controller/main.go:	releaseNamespace := o.ReleaseNamespaces[0]
cmd/release-controller/main.go:	for _, ns := range o.ReleaseNamespaces {
```

It's also not clear to me what the difference between `name` and `releaseNamespace` is, if we can pass the same value to both of them.  Can `NewExecReleaseFiles` grow a godoc comment?

Reply:
I have cleaned up the preexisting pattern, added a couple godoc comments, and refactored the annotation/labels to be consistently named with the files-cache